src/modules/Microsoft.Resources/deploymentScripts/deploy.bicep

```diff
@@ -0,0 +1,139 @@
+@description('Required. Display name of the script to be run.')
+param name string
+
+@description('Optional. The ID(s) to assign to the resource.')
+param userAssignedIdentities object = {}
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Type of the script. AzurePowerShell, AzureCLI.')
+@allowed([
+  'AzurePowerShell'
```

Comment:
Since we're using PowerShell for everything else, should we standardize and remove this option?

Reply:
+1 on standardization